user-guide/Reference/DataMiner_Tools/Spectrum_Simulation_Tool.md

```diff
@@ -6,38 +6,14 @@ uid: Spectrum_Simulation_protocol
 
 To test DataMiner Spectrum Analysis, a spectrum simulation protocol is available. You can for example use it to verify the “set parameter” commands that SLSpectrum sends to the protocol, or to get a trace on the screen and test the various features of the spectrum user interface and spectrum monitoring without a real device.
```

Comment:
Would add an extra sentence: Furthermore, you can simulate various carriers and amplitudes that closely resemble real-life use cases.